clingwrapper/src/clingwrapper.cxx

```diff
@@ -1523,32 +1528,70 @@ Cppyy::TCppMethod_t Cppyy::GetMethodTemplate(
 
 }
 
-//
-// static inline
-// std::string type_remap(const std::string& n1, const std::string& n2)
-// {
-// // Operator lookups of (C++ string, Python str) should succeed for the combos of
-// // string/str, wstring/str, string/unicode and wstring/unicode; since C++ does not have a
-// // operator+(std::string, std::wstring), we'll have to look up the same type and rely on
-// // the converters in CPyCppyy/_cppyy.
-//     if (n1 == "str" || n1 == "unicode") {
-//         if (n2 == "std::basic_string<wchar_t,std::char_traits<wchar_t>,std::allocator<wchar_t> >")
-//             return n2;                      // match like for like
-//         return "std::string";               // probably best bet
-//     } else if (n1 == "float") {
-//         return "double";                    // debatable, but probably intended
-//     } else if (n1 == "complex") {
-//         return "std::complex<double>";
-//     }
-//     return n1;
-// }
+static inline std::string type_remap(const std::string& n1,
+                                     const std::string& n2) {
+    // Operator lookups of (C++ string, Python str) should succeed for the
+    // combos of string/str, wstring/str, string/unicode and wstring/unicode;
+    // since C++ does not have a operator+(std::string, std::wstring), we'll
+    // have to look up the same type and rely on the converters in
+    // CPyCppyy/_cppyy.
+    if (n1 == "str" || n1 == "unicode") {
+        // if (n2 ==
+        // "std::basic_string<wchar_t,std::char_traits<wchar_t>,std::allocator<wchar_t>
+        // >")
+        //     return n2;                      // match like for like
+        return "std::string"; // probably best bet
+    } else if (n1 == "float") {
+        return "double"; // debatable, but probably intended
+    } else if (n1 == "complex") {
+        return "std::complex<double>";
+    }
+    return n1;
+}
+
+void Cppyy::GetClassOperators(Cppyy::TCppScope_t klass,
+                              const std::string& opname,
+                              std::vector<TCppScope_t>& operators) {
+    if (opname == "operator+")
```

Comment:
This is something we can fold into an API: `const char * GetOperatorKindAsString(Cpp::OperatorKind K);`

Reply:
For this use case, shouldn't it be `Cpp::OperatorKind Cpp::GetOperatorKindFromString(std::string op)`?
Also, I believe this should be part of InterOp.
Should I add both? Any thoughts?

Reply:
You are right. Let's keep it as is.